cfme/tests/v2v/test_csv_import.py

```diff
@@ -0,0 +1,182 @@
+    plan_view.vms.hidden_field.fill(temp_file.name)
+    wait_for(lambda: plan_view.vms.table.is_displayed,
+             timeout=60, message='Wait for VMs view', delay=2)
+    plan_view.vms.table[0][1].widget.click()
+    error_msg = "Empty name specified"
+    assert error_msg == plan_view.vms.popover_text.read()
+    plan_view.cancel_btn.click()
+
+
+def test_csv_invalid_vm(appliance, infra_map):
+    """Test csv with invalid vm name"""
+    plan_view = migration_plan(appliance, infra_map)
+    temp_file = tempfile.NamedTemporaryFile(suffix='.csv')
+    with open(temp_file.name, 'w') as f:
+        f.write("Name\n{}".format(fauxfactory.gen_alpha(10)))
+    plan_view.vms.hidden_field.fill(temp_file.name)
+    wait_for(lambda: plan_view.vms.table.is_displayed,
+             timeout=60, message='Wait for VMs view', delay=2)
+    plan_view.vms.table[0][1].widget.click()
+    error_msg = "VM does not exist"
+    assert error_msg == plan_view.vms.popover_text.read()
+    plan_view.cancel_btn.click()
+
+
+def test_csv_valid_vm(appliance, infra_map, valid_vm):
+    """Test csv with valid vm name"""
+    plan_view = migration_plan(appliance, infra_map)
+    temp_file = tempfile.NamedTemporaryFile(suffix='.csv')
+    with open(temp_file.name, 'w') as f:
+        f.write("Name\n{}".format(valid_vm))
+    plan_view.vms.hidden_field.fill(temp_file.name)
+    wait_for(lambda: plan_view.vms.table.is_displayed,
+             timeout=60, message='Wait for VMs view', delay=2)
+    plan_view.vms.table[0][1].widget.click()
+    error_msg = "VM available for migration"
+    assert error_msg == plan_view.vms.popover_text.read()
+    plan_view.cancel_btn.click()
+
+
+def test_csv_duplicate_vm(appliance, infra_map, valid_vm):
+    """Test csv with duplicate vm name"""
+    plan_view = migration_plan(appliance, infra_map)
+    temp_file = tempfile.NamedTemporaryFile(suffix='.csv')
+    with open(temp_file.name, 'w') as f:
+        f.write("Name\n{}\n{}".format(valid_vm, valid_vm))
+    plan_view.vms.hidden_field.fill(temp_file.name)
+    wait_for(lambda: plan_view.vms.table.is_displayed,
+             timeout=60, message='Wait for VMs view', delay=2)
+    plan_view.vms.table[2][1].widget.click()
+    error_msg = "Duplicate VM"
+    assert error_msg == plan_view.vms.popover_text.read()
+    plan_view.cancel_btn.click()
+
+
+def test_csv_archived_vm(appliance, infra_map, archived_vm):
+    """Test csv with archived vm name"""
+    plan_view = migration_plan(appliance, infra_map)
+    temp_file = tempfile.NamedTemporaryFile(suffix='.csv')
+    with open(temp_file.name, 'w') as f:
+        f.write("Name\n{}".format(archived_vm))
+    plan_view.vms.hidden_field.fill(temp_file.name)
+    wait_for(lambda: plan_view.vms.table.is_displayed,
+             timeout=60, message='Wait for VMs view', delay=2)
+    plan_view.vms.table[0][1].widget.click()
+    error_msg = "VM is inactive"
+    assert error_msg == plan_view.vms.popover_text.read()
+    plan_view.cancel_btn.click()
```

Comment:
**REQUIRED**
These tests look similar. Please consider to create a new fixture or parametrize tests to avoid self repeating.